project/app/src/main/java/com/example/cmput301w21t25/experiments/Experiment.java

```diff
@@ -35,111 +37,92 @@
  * this class is an Abstract class for experiments
  * it sets up the most important methods for the classes that the children will inherit
  */
-public abstract class Experiment implements Serializable {
+public class Experiment implements Serializable {
 
     /****************************************
                     ATTRIBUTES
     ****************************************/
-    private String owner;
-    private String ownerID;
-    public String name;
-    private String description;
-    private String type;
-    private ArrayList<String> tags;
-    private int minNumTrials;
-    private int currentNumTrials;
+
     private boolean isPublished = false;
+    private boolean published = false;
+
     private String fb_id = "";
-    private Date date;
+    
+    private boolean geoEnabled;
     private boolean isEnded = false;
-
-    private ArrayList<User> subscribedUsers = new ArrayList<User>(); //all currently subscribed users
-    private ArrayList<User> allUsers = new ArrayList<User>(); //for users that were subscribed, added data, and unsubscribed
+    private int minNumTrials;
+    
+    public String name;
+    private String type;
+    private String description;
+    
+    //Should we just be passing in a User object that has a name and ID associated with it?
```

Comment:
Should we just be passing in a User object that has a name and ID associated with it?
> this comment not really for this PR, more just a thought for in general, doesn't need to be changed right now